src/guide/receive_payment.md

```diff
@@ -1,8 +1,177 @@
-# Receiving Lightning payments
+# Receiving Payments
 
 With the Breez Liquid SDK you aren't required to open a channel and set up your inbound liquidity.
 
-Once the SDK is initialized, you can directly begin receiving payments.
+Once the SDK is initialized, you can directly begin receiving payments. The receive process takes two steps:
+1. [Preparing the Payment](receive_payment.md#preparing-payments)
+1. [Receiving the Payment](receive_payment.md#receiving-payments-1)
+
+## Preparing Payments
+During the prepare step, the SDK ensures that the inputs are valid with respect to the specified payment method,
+and also returns the relative fees related to the payment so they can be confirmed.
+
+
+The SDK currently supports three methods of receiving: Lightning, Liquid and Cross-chain (Bitcoin):
```

Comment:
Can we change to ordering? I also think "cross-chain" is not needed, it's an SDK implementation detail
```suggestion
The SDK currently supports three methods of receiving: Lightning, Bitcoin and Liquid:
```